src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -188,4 +188,10 @@
   <data name="Arg_PathEmpty" xml:space="preserve">
     <value>The path is empty.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedPosNum" xml:space="preserve">
+    <value>Positive number required.</value>
+  </data>
+  <data name="Argument_InvalidFileModeAndAccessCombo" xml:space="preserve">
```

Comment:
@JeremyKuhne 
I was testing some combinations of parameters, and found that .NET Framework throws this exception if you combine `FileSystemRights.ReadData` with `FileMode.Create`. Here's the code that does that check:
https://referencesource.microsoft.com/#mscorlib/system/io/filestream.cs,676

I am currently not doing these combination checks here. Should I throw in the new method if we encounter these combinations?

Reply:
For the combos, yes.

Reply:
Done. I'll add a test too.